changelog/9502.improvement.rst

```diff
@@ -0,0 +1 @@
+Added an environment variable which helps to check if a code is running from within a pytest run
```

Comment:
```suggestion
Added :envvar:`PYTEST_VERSION` environment variable which is defined at the start of the pytest session and undefined afterwards. It contains the value of ``pytest.__version__``, and among other things can be used to easily check if code is running from within a pytest run.
```

Besides this, please also add a section for it in `doc/en/reference/reference.rst` in the `Environment Variables` section.